packages/terra-dialog/docs/README.md

```diff
@@ -13,9 +13,22 @@ Dialogs are temporary views that can be used in a myriad of ways. Dialogs have t
 import React from 'react';
 import Dialog from 'terra-dialog';
 
-<Dialog header="Header Content" footer="Footer Content">some body content</Dialog>
+const header = 'Header Content';
+
+const footer = <Dialog.Footer>Footer Content</Dialog.Footer>;
+
+const body = (<Dialog.Body>
+  <p>This is my body content.</p>
+  <p>This is some more content.</p>
+</Dialog.Body>);
+
+<Dialog header={header} footer={footer} onClose={() => alert('Close')} >
+  {body}
+</Dialog>
 ```
 
+Note: This component has an i18n dependency that requires setup per [these steps](https://github.com/cerner/terra-core/#internationalization-i18n).
```

Comment:
When trying to get ActionHeader working, I ran into issues because I didn't realize the need for Base as an ancestor. I wanted to include a link and brief description here to help future consumers avoid this confusion, too. Any tips on better wording appreciated.